### PR TITLE
Issue #SB-19488 fix: Added content routes for whitelisting

### PR DESCRIPTION
**Pull request by rajeshkumaravel**
- Issue #SB-19488 fix: Added content routes for whitelisting

### CODE REVIEW

File: src/app/helpers/apiWhiteList.js [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+ * 2. If API is does not exists then request is terminated and 401 UNAUTHORIZED response is sent
+ * @since release-3.1.0
+ */
+const isAPIWhiteListed = () => {
+  return function (req, res, next) {
+    const REQ_URL = req.originalUrl;
+    if (_.get(ROUTE, REQ_URL)) {
+      next();
+    } else {
+      logger.error({
```
Comment (vinukumar-vs) [inline src/app/helpers/apiWhiteList.js:26]: Use UtilService.logError method for standard portal log format

File: src/app/helpers/apiWhiteList.js [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+const dateFormat  = require('dateformat');
+const ROUTE       = require('./whitelistApis');
+
+/**
+ * @description - Function to check whether incoming API is whitelisted (or) not
+ * 1. If incoming API is present in API_LISTS (whitelistApis); then request is passed further
+ * 2. If API is does not exists then request is terminated and 401 UNAUTHORIZED response is sent
+ * @since release-3.1.0
+ */
+const isAPIWhiteListed = () => {
```
Comment (vinukumar-vs) [inline src/app/helpers/apiWhiteList.js:20]: Change the name to isValid or IsExist

File: src/app/routes/contentRoutes.js [hunk 3]
```diff
@@ -37,6 +38,7 @@ module.exports = (app) => {
         }))
 
     app.all('/content/*',
+        isAPIWhitelisted.isAPIWhiteListed(),
```
Comment (vinukumar-vs) [inline src/app/routes/contentRoutes.js:41]: after naming changes, it should be like
apiWhiteList.isValid()

File: src/app/helpers/apiWhiteList.js (hunk 2 above)
Comment (rajeshkumaravel) [reply src/app/helpers/apiWhiteList.js:20]: I would recommend having method name as **isAllowed**; which conveys whether API is allowed or not.
Please confirm on this.
_P.S. Awaiting response._

File: src/app/helpers/apiWhiteList.js (hunk 1 above)
Comment (rajeshkumaravel) [reply src/app/helpers/apiWhiteList.js:26]: As a reference from whitelistUrl.js the logger used is ```const logger = require('sb_logger_util_v2');```
And also the UtilService.logError is nowhere used;

Comment (vinukumar-vs) [reply src/app/helpers/apiWhiteList.js:26]: Use logError method going foward.

Comment (rajeshkumaravel) [reply src/app/helpers/apiWhiteList.js]: Changes updated.

File: src/app/helpers/apiWhiteList.js (hunk 2 above)
Comment (rajeshkumaravel) [reply src/app/helpers/apiWhiteList.js]: Changes updated.

**Pull request merged**